### ISSUE TITLE
False positive for unused variable with typed properties with default values.

### ISSUE THREAD

**Issue by twistor** [slevomat/coding-standard#1180]
```php
class A
{
    protected bool $foo = false;

    public function getFoo(): bool
    {
         return $this->foo;
    }

    public function setFoo(): void
    {
        $this->foo = true;
    }
}
```

Results in "Unused variable $foo."

**Comment by kukulich** [comment]
@twistor What version do you use? Have you tried `dev-master` ?

**Comment by twistor** [comment]
Darn I thought I had upgraded but apparently it didn't stick. It works. Thanks!
